library/Freckle/App/Database.hs

```diff
@@ -57,6 +62,9 @@ class HasSqlPool app where
 instance HasSqlPool SqlPool where
   getSqlPool = id
 
+class HasActiveConnectionGauge app where
+  getActiveConnectionGauge :: app -> Gauge.Gauge
```

Comment:
Is there a case where we are okay with this failing? E.G. not having a gauge, or should we just expect all our apps to monitor their active connections?

Reply:
For ease of implementation elsewhere I could definitely see this as being a `Maybe Gauge`, those as below I'm uncertain what the architecture requirements are intended to be.

Reply:
I'd go with `Maybe` to make this optional.

The goal is `runDB` should be able to Do All The Things but be easy to use, so we naturally use it consistently. So if you don't need/want the metrics, that should still be an easy use-case.

The same applies to `HasStatsClient`, but (I believe) that module provides a `DummyClient` that can be used to opt-out but still provide the interface.